CHANGELOG.md

```diff
@@ -39,7 +38,6 @@
 - Add documentation for copy, cut, and paste blocks in Volto. @MAX-786 [#3827](https://github.com/plone/volto/issues/3827)
 - Fixed Grammar error @SaiRev0 [#4272](https://github.com/plone/volto/issues/4272)
 
-
 ## 16.9.0 (2023-01-27)
```

Comment:
Why all the whitespace changes? Please do not edit this file as part of this PR. If any changes are needed to this file, then a separate PR should be done to handle those changes.